frontend/packages/knative-plugin/integration-tests/features/admin-perspective/eventing-page-admin.feature

```diff
@@ -33,12 +35,26 @@ Feature: Eventing page at Administrator perspective
              Then user will be redirected to Project Details page
               And user will see channel created
 
+        @regression @to-do
+        Scenario: Adding new Trigger
+            Given user is at eventing page
+              And user has created knative service "hello-openshift"
```

Comment:
We can create the Trigger, within the "aut-eventing-page" namespace, Are you still looking for different namespace